packages/data-store/src/uniformCredential/AbstractUniformCredentialStore.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  AddUniformCredentialArgs,
+  GetUniformCredentialArgs,
+  GetUniformCredentialsArgs,
+  RemoveUniformCredentialArgs,
+  UpdateUniformCredentialStateArgs,
+} from '../types/uniformCredential/IAbstractUniformCredentialStore'
+import { UniformCredentialEntity } from '../entities/uniformCredential/UniformCredentialEntity'
+
+export abstract class AbstractUniformCredentialStore {
+  abstract getUniformCredential(args: GetUniformCredentialArgs): Promise<UniformCredentialEntity>
+  abstract getUniformCredentials(args?: GetUniformCredentialsArgs): Promise<Array<UniformCredentialEntity>>
```

Comment:
We need to add pagination support, given we know we will need this

Reply:
added pagination and sorting to the parameters of get many method